packages/hoppscotch-common/src/helpers/import-export/import/hoppEnv.ts

```diff
@@ -17,15 +17,33 @@ const hoppEnvSchema = z.object({
   ),
 })
 
-export const hoppEnvImporter = (content: string) => {
-  const parsedContent = safeParseJSON(content, true)
+type HoppEnv = z.infer<typeof hoppEnvSchema>
 
-  // parse json from the environments string
-  if (O.isNone(parsedContent)) {
+export const hoppEnvImporter = (contents: string[]) => {
+  const parsedContents = contents.map((str) => safeParseJSON(str, true))
```

Comment:
Exporting a single environment takes the shape of an object. Hence, we convert all the entries to an array, ending up with a collection of arrays flattened at a later stage after processing.